Packs/ProofpointTAP/Integrations/ProofpointTAP_v2/ProofpointTAP_v2.py

```diff
@@ -51,7 +51,7 @@ def get_fetch_times(last_fetch):
         last_fetch = datetime.strptime(last_fetch, time_format)
     elif isinstance(last_fetch, datetime):
         times.append(last_fetch.strftime(time_format))
-    while now - last_fetch > timedelta(minutes=59):
+    while now - last_fetch > timedelta(minutes=59) and now - last_fetch > timedelta(minutes=60):
```

Comment:
why add another condition? The first condition is included in the second.
Also, why does this solve the bug?